build/source/engine/systemSolv.f90

```diff
@@ -217,12 +217,16 @@ real(dp)                        :: fScale(nState)                ! characteristi
  real(qp)                        :: rVec(nState)    ! NOTE: qp    ! residual vector
  real(dp)                        :: rAdd(nState)                  ! additional terms in the residual vector
  real(dp)                        :: fOld,fNew                     ! function values (-); NOTE: dimensionless because scaled
- real(dp)                        :: xMin,xMax                     ! state minimum and maximum (mixed units) 
+ real(dp)                        :: xMin,xMax                     ! state minimum and maximum (mixed units)
```

Comment:
This is just a change in white space. Please ensure that all code has been run through a white space trimmer before review.